lib/beaker/dsl/install_utils.rb

```diff
@@ -901,14 +905,21 @@ def install_puppet_from_msi( host, opts )
           dest = "C:\\Windows\\Temp\\#{host['dist']}.msi"
 
           on host, "set PATH=\"%PATH%;#{host['puppetbindir']}\""
-          on host, "setx PATH \"%PATH%;#{host['puppetbindir']}\""
+
+          bindir = host['puppetbindir'].gsub(/"/,'')
+
+          on host, "setx PATH \"%PATH%;#{bindir}\""
```

Comment:
You an now do this through host.add_env_var.  Also, it may already have been during the configuration/validation stage of the run.